src/testbench.h

```diff
@@ -143,7 +143,6 @@ public slots:
 
         case 4: // PROTMESSID_CONN_CLIENTS_LIST
             vecChanInfo[0].iChanID = GenRandomIntInRange ( -2, 20 );
-            vecChanInfo[0].iIpAddr = GenRandomIPv4Address().toIPv4Address();
```

Comment:
I _think_ I'd like these to have explicit zero put in, in place of the ipAddr.  (Same at old 281 below.)

Reply:
Explicit zero put into where? The point of this change is the removal of the `iIpAddr` class member.

Reply:
Sorry, code overload...